src/Sylius/Behat/Page/Shop/HomePage.php

```diff
@@ -105,6 +105,16 @@ public function switchLocale($localeCode)
     /**
      * {@inheritdoc}
      */
+    public function countProductsItems()
+    {
+        $products = $this->getDocument()->findAll('css','.sylius-product-name');
+
+        return count($products);
```

Comment:
`findAll` returns `NodeElement[]` according to its definition, so we shouldn't be bothered by that method returning uncountable value.